scripts/chemistry/periodic_table/README.md

```diff
@@ -0,0 +1,28 @@
+# Periodic table of elements
```

Comment:
Is pulling this in intentional? Preferably, this is a separate PR

Reply:
Sorry, all chemistry/* files are in another PR. My master branch seems corrupted. I'll remove these files.

Reply:
ok, thanks!